### PR TITLE
Revised externalexamples/readme.txt for clairity and conciseness.

### LINKED ISSUE #1015: Rewrite externalexamples/readme.txt
externalexamples/readme.txt is weakly worded and somewhat clunky in its reading.

### PR DESCRIPTION
**Pull request by anTrodaire**
As part of a technical writing course, we were asked to improve
documentation on GitHub. I've revised externalexamples/readme.txt so it
reads better.

Fixes #1015

### CODE REVIEW

File: externalexamples/readme.txt [hunk 1]
```diff
@@ -1,7 +1,6 @@
-This directory is intended to contain code for projects that use Umple, but are not
+This directory is intended to contain code for projects that use Umple but are not
 part of the Umple project itself.
 
-We invite people to create their own umple examples, with proper test cases included.
-We can then build them automatically. That way we can help ensure that as we update
-the compiler it doesn't break your code, and also your code serves as additional
-test cases for us.
+We invite people to create their own umple exaples, including proper test cases.
```
Comment (vahdat-ab) [inline externalexamples/readme.txt]: Please consider the following cases,
- Umple should be started always with an uppercase letter `U`
- There is a typo in your modification  -> `exaples`

**Review by vahdat-ab — CHANGES_REQUESTED** [review]

**Comment by vahdat-ab** [comment]
when you do PR, please make sure it points at the issue you created. Therefore, once the PR is merged, teh issue will be closed.

File: externalexamples/readme.txt (hunk 1 above)
Comment (anTrodaire) [reply externalexamples/readme.txt]: Sorry about that! Fixed.

**Comment by vahdat-ab** [comment]
Thank you for the PR 👍

**Pull request merged**